pkgs/tools/misc/fastfetch/default.nix

```
@@ -0,0 +1,81 @@
+{ chafa
+, cmake
+, dbus
+, dconf
+, fetchFromGitHub
+, glib
+, imagemagick_light
+, lib
+, libglvnd
+, libxcb
+, makeWrapper
+, ocl-icd
+, opencl-headers
+, pciutils
+, pkg-config
+, stdenv
+, vulkan-loader
+, wayland
+, xfce
+, xorg
+, zlib
+, enableChafa ? false
+, enableImageMagick ? false
+, enableOpenCLModule ? true
+, enableOpenGLModule ? true
+, enableVulkanModule ? true
+, enableWayland ? true
+, enableX11 ? true
+, enableXFCE ? false
+}:
+
+stdenv.mkDerivation rec {
+  pname = "fastfetch";
+  version = "1.5.6";
+
+  src = fetchFromGitHub {
+    owner = "LinusDierheimer";
+    repo = pname;
+    rev = version;
+    hash = "sha256-OsK78irfO9r6pSEmlN6mFoKhQZgBqIYO8syj5AHH4IE=";
+  };
+
+  nativeBuildInputs = [ cmake makeWrapper pkg-config ];
+
+  runtimeDependencies =
+    [
+      dbus
+      dconf
+      glib
+      pciutils
+      zlib
+    ]
+    ++ lib.optionals (enableChafa) [ chafa ]
```

The brackets around `enableChafa` don't do anything here. Also, you can use `lib.optional` instead so you don't need to create a list.
```suggestion
    ++ lib.optional enableChafa chafa
```
same goes for the others.